content/millennium/r4/scheduling/appointment.md

```diff
@@ -161,6 +161,7 @@ _Implementation Notes_
 * Only operations on the paths listed below are supported.
 * For video visit link patch operation paths, `contained` index 0 represents the provider link and `contained` index 1 represents the patient link.
 * Video Visit linking requires additional client configuration and application support.
```

Comment:
formatting nit: We're pretty inconsistent with casing of 'Video Visit' throughout this page. All rendered text of `video visit` should be changed to `Video Visit`.  That applies to both the markdown files and yaml

Reply:
adjusted to Video Visit throughout https://github.com/cerner/fhir.cerner.com/pull/385/commits/ab0f399f4c56a6788c04ccd244ad0a9e9453ede9#